### ISSUE TITLE
import read10xVisium function

### ISSUE THREAD

**Issue by drighelli** [waldronlab/TENxIO#3]
As already discussed multiple times with @LiNk-NY and in some other issues around, it would be nice to have all the import functions into a unique package. We think this would be a nice container for all of them.

So, Marcel asked me to import the `read10xVisium` function into this package to remove dependencies like dropletUtils from our SpatialExperiment class so I'm linking @lmweber and @HelenaLC to see if they agree with this process

**Comment by lmweber** [comment]
Yes, I think this makes sense to me, so there would be more of a separation between the core class (`SpatialExperiment`) and specific helper / loading functions that are only relevant for for certain platforms (`read10xVisium()`).

This would then also solve the `DropletUtils` dependencies issue in [#139](https://github.com/drighelli/SpatialExperiment/issues/139).

Is the plan also to import `read10xCounts()` from `DropletUtils` into the `TENxIO` package? Either way it would be good to have consistency between how `read10xCounts()` and `read10xVisium()` are handled.
